README.md

```diff
@@ -682,6 +682,7 @@ Here are some companies/projects (alphabetical order) using OpenAPI Generator in
 - [Yelp](https://www.yelp.com/)
 - [Zalando](https://www.zalando.com)
 - [3DS Outscale](https://www.outscale.com/)
+- [Deeporute.ai](https://www.deeproute.ai/)
```

Comment:
Can please you add it to the list based on alphabetical order?

Reply:
copy that